cms/static/js/views/pages/container.js

```diff
@@ -162,6 +170,16 @@ define(["jquery", "underscore", "gettext", "js/views/pages/base_page", "js/views
                 }
             },
 
+            updatePreviewButton: function(show_previews){
```

Comment:
updatePreviewButton concerns the container page rather than the container view-- which is only the area holding the XBlocks.

I've moved this into the paged container page rather than the normal one, though.